src/app.ts

```diff
@@ -79,8 +79,8 @@ async function setupMajorScraperCron() {
 }
 
 async function setupMatchingCron() {
-  // Match users with one another every 2 minutes (testing purposes - will be weekly soon)
-  cron.schedule('*/2 * * * *', async () => {
+  // Match users with one another every 10 minutes (testing purposes - will be weekly soon)
+  cron.schedule('*/10 * * * *', async () => {
     matcher();
   });
```

Comment:
maybe this would be a good place to return a diff schedule depending on the `NODE_ENV` set to prod or dev?